server/swagger_v2.yml

```diff
@@ -0,0 +1,482 @@
+openapi: "3.0.0"
+info:
+  title: Gateway Service
+  version: 0.1.0
+servers:
+  - url: /chronograf/v2
+paths:
+  /cells:
+    post:
+      tags:
+        - Cells
+      summary: Create a cell
+      requestBody:
+          description: cell to create
+          required: true
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Cell"
+      responses:
+        '201':
+          description: Added cell
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Cell"
+        default:
+          description: unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+    get:
+      tags:
+        - Cells
+      summary: Get all cells
+      parameters:
+        - in: query
```

Comment:
Org is not in the implementation. Should it?

Reply:
nope not yet